README.md

```diff
@@ -60,7 +60,7 @@ La web de **Juguetear** surge como respuesta a la necesitad de modernizar y mejo
 Todo el contenido se va a administrar usando Sanity como [Headless CMS](https://www.sanity.io/headless-cms). Para poder autenticarte y acceder al contenido (juguetes, textos, fotos, etc.), tenés que seguir los siguientes pasos:
 
 1. En el archivo `.env` (si no lo tenés aún, hay que crearlo en el directorio raíz) tenés que agregar los _token_ necesarios. Podes mirar el archivo `.env.example` para saber cuales son.
-2. Una vez guardado el archivo, si tenés el server ya iniciado, vas a tener que reiniciarlo. Para eso, tenés que ir a la consola de comandos, apretar ctrl + C, y luego ejecutar `npm run dev`.
+2. Una vez guardado el archivo, si tenés el server ya iniciado, vas a tener que reiniciarlo. Para eso, tenés que ir a la consola de comandos, apretar ctrl + C, y luego ejecutar `yarn run dev` o directamente `yarn dev` que en escencia, hacen lo mismo.
```

Comment:
Hola @brahianpdev !
Un pequeño detalle acá, es `esencia`.
Así mismo, creo que podrías hacerlo un poco más simple "... y luego ejecutar `yarn dev` / `yarn run dev`."

Reply:
Concido con lo que dice @LPRegen! Que sea lo más sencillo y directo posible.

Reply:
Hello @LPRegen @nykka, esto les comente por Discord el otro dia, esta parte del readme se podria cambiar..

- En el primer paso no haria falta si agregamos los "tokens" a la config de Sanity directamente en #85.
- En el segundo paso no haria falta reiniciar el server porque Nextjs 12.3+ incluye [improved-fast-refresh](https://nextjs.org/blog/next-12-3#improved-fast-refresh).
>With Next.js 12.3, some configuration files now also hot reloaded when making edits, including:
.env and env.* variants
jsconfig.json
tsconfig.json
You now do not need to restart your local development server when making changes to these configuration files.

Thoughts?

Reply:
@TonyMckes gracias por la aclaración! Entonces hay que volar toda esta parte @brahianpdev.
Hay que borrar desde: "Para poder autenticarte y acceder al contenido..." hasta "...y luego ejecutar npm run dev."